README.md

```diff
@@ -64,16 +64,16 @@ ai init
 ```
 
 - This will first prompt to you to login to Azure
-- Then it will ask you to select or create resources, choose  **New Azure AI Project** and follow the prompts to create an:
-   - Azure AI resource
+- Then it will ask you to select or create resources. Select **New AI Project** > **(Create w/ integrated Open AI + AI Services)** and follow the prompts to create an:
+   - Azure AI hub resource
```

Comment:
If this is supposed to represent what the CLI prints out, it doesn't print out "hub" ... Don't make this change if you want to match currently released CLI.

Reply:
please verify that these match what the CLI actually prints.